docs/State-checkpointing.md

```diff
@@ -94,6 +94,9 @@ is saved and then the checkpoint tuple is forwarded to the next component. Each
 streams before it saves its state so that the state represents a consistent state across the topology. Once the checkpoint spout receives
 ACK from all the bolts, the state commit is complete and the transaction is recorded as committed by the checkpoint spout.
 
+The state checkpointing does not currently checkpoint the state of the spout. Yet, once the states of all bolts are checkpointed and once the checkpoint tuples is acked, the tuples emitted by the spout are acked. 
+It also implies that `topology.state.checkpoint.interval.ms` is lower than `topology.message.timeout.secs`. 
```

Comment:
once the *STATE* of all the bolts are checkpointed, and once the checkpoint tuples *ARE* acked.... by the spout are *ALSO* acked.